src/components/Game.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+    <v-layout>
+        <v-flex xs12 sm6 offset-sm3>
+            <v-layout align-center>
+                <v-flex>
+                    <h1 v-if="isLoading">Loading...</h1>
+                    <h1 v-if="questionsNotFound">Questions not found!</h1>
+                    <h1 v-if="isComplete">Game complete! ({{scoreLabel}})</h1>
+                    <h1 v-else>{{scoreLabel}}</h1>
+                </v-flex>
+            </v-layout>
+            <v-progress-linear v-model="gameProgress"></v-progress-linear>
+            <v-card color="blue-grey darken-2" class="white--text" v-if="!questionsNotFound && !isComplete">
+                <v-card-title primary-title>
+                    <h3 class="headline">{{currentQuestion.question}}</h3>
+                </v-card-title>
+                <v-card-text>
+                    <v-container fluid grid-list-md>
+                        <v-layout row wrap>
+                            <v-flex xs6 true v-for="(alternative, index) in currentQuestion.alternatives" :key="index">
+                                <v-card height="100%" @click.native="answer(currentQuestion, index)" dark>
+                                    <v-card-text>
+                                        {{alternative}}
+                                    </v-card-text>
+                                </v-card>
+                            </v-flex>
+                        </v-layout>
+                    </v-container>
+                </v-card-text>
+            </v-card>
+        </v-flex>
+    </v-layout>
+</template>
+<script>
+import { db } from '@/firebase'
+
+export default {
+  name: 'game',
+
+  data() {
+    return {
+      questions: [],
+      currentQuestion: {},
+      currentIndex: 0,
+      isLoading: true,
+      isComplete: false,
+      questionsNotFound: false,
+      gameProgress: 0,
+      gameScore: 0,
+      gameSize: 0
+    }
+  },
+
+  created() {
+    this.bindDbRefToQuestions(`questions`)
+  },
+
+  watch: {
+    $route(to) {
```

Comment:
We probably don't need to watch for url changes here, yet.
Maybe in the future we will have games with their own ids. Then we could watch for url parameter changes to display the right question, eg:
`opquiz.se/#/<game-id>/1`   (question 1 in that game)
Watching for changes would allow us (and users) to change to, for example,
`opquiz.se/#/<game-id>/5`   (question 5 in that game)

Maybe desirable, maybe not!? :)

Reply:
Så vi skulle kunna skippa watch helt här? Var för länge sen jag skrev den så kommer inte ihåg om jag hade nån intelligent tanke...